src/test/scala/com/eclipsesource/schema/FormatSpec.scala

```diff
@@ -63,28 +63,49 @@ class FormatSpec extends Specification with JsonSpec {
     }
 
     "validate int32 format" in {
-      val schema = JsonSource.schemaFromString(
+      val integerSchema = JsonSource.schemaFromString(
+        """
+          |{
+          |  "type": "integer",
+          |  "format": "int32"
+          | }
+        """.stripMargin).get
+      val numberSchema = JsonSource.schemaFromString(
```

Comment:
Yes, it is kind of what I do in my product. I do not know how to verify this PR in my product without having a package pulled as a dependency. If you can educate me how it could be done, I can try.